Makefile

```diff
@@ -12,7 +13,7 @@ setup:
 	@pip install -e .[tests]
```

Comment:
Do you think it should be done the same way for pip3?

Reply:
Fixed in the follow-up commit